stories/Molecules/CustomSelect/CustomSelect.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import './customselect.scss';
+import init from '../../assets/js/select-init';
+
+const CustomSelect = ({ text }) => {
+  useEffect(() => {
+    init();
+  }, []);
+  return (
+    <div>
```

Comment:
@Ashish-srij Please remove this extra div

Reply:
Done